api/src/main/java/org/openmrs/module/fhir2/providers/r3/PractitionerFhirResourceProvider.java

```diff
@@ -55,6 +59,38 @@ public Practitioner getPractitionerById(@IdParam @NotNull IdType id) {
 		return Practitioner30_40.convertPractitioner(practitioner);
 	}
 	
+
+	@Create
+	public MethodOutcome createPractitioner(@ResourceParam Practitioner practitioner) {
+		return FhirProviderUtils.buildCreate(practitionerService.create(Practitioner30_40.convertPractitioner(practitioner)));
+	} 
+	
+	@Update
+	@SuppressWarnings("unused")
```

Comment:
Why is there a need to use this annotation ? You have anyways used these methods in your unit tests. Please find this useful throughout the PR.

Reply:
i wanted to maintain the status quo as in all other resources